### PR TITLE
Use xeval from deno_std

### PR DESCRIPTION
**Pull request by nayeemrmn**

### CODE REVIEW

File: cli/tests/integration_tests.rs [hunk 1]
```diff
@@ -222,24 +222,6 @@ itest!(_029_eval {
   output: "029_eval.out",
 });
 
-itest!(_030_xeval {
-  args: "xeval console.log($.toUpperCase())",
```
Comment (hayd) [inline cli/tests/integration_tests.rs]: aren't the tests still valuable?

Comment (ry) [reply cli/tests/integration_tests.rs]: I agree - even though deno_std has unit tests, I think the fact that xeval is a subcommand justifies having these integration tests.

Comment (kevinkassimo) [reply cli/tests/integration_tests.rs]: +1 for keeping the test -- this also ensures we do not have unexpected problems e.g. API change caused issues when upgrading std deps

Comment (nayeemrmn) [reply cli/tests/integration_tests.rs]: 👍 I left out 043, one for each arg is sufficient here.

**Pull request merged**